doc/governance/governance.rst

```diff
@@ -0,0 +1,124 @@
+
+.. _Governance:
+
+Project Governance
+==================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Since 2022, the ros-controls project has been governed by the `Open Source Robotics Alliance (OSRA) <https://osralliance.org/>`__.
```

Comment:
```suggestion
Since 2025, the ros-controls project has been governed by the `Open Source Robotics Alliance (OSRA) <https://osralliance.org/>`__.
```